.ko.yaml

```diff
@@ -0,0 +1,2 @@
+baseImageOverrides:
+  github.com/tektoncd/pipeline/cmd/resolvers: cgr.dev/chainguard/git@sha256:566235a8ef752f37d285042ee05fc37dbb04293e50f116a231984080fb835693
```

Comment:
I think there might be some changes to be done in the `tekton/publish.yaml` task as well.

Reply:
In addition to modifying the task's inline .koconfig like [is already done](https://github.com/tektoncd/pipeline/pull/8677/files#diff-ca8a76c3e402ca5269339af1eab8704146bc108d3270637f4642a1d4c5bdf465)?